### PR TITLE
Part 2: Doxygen; data member names; remove "value" from Config UI; (for many Sensors) 

### PR DESCRIPTION
**Pull request by ba58smith**
Continuing the cleanup:
- Sensor base class
- SHT31
- SystemInfo
- Ultrasonic

Ready to review. (Just 3 more sensors to go!)

### CODE REVIEW

File: src/sensors/sht31.h [hunk 1]
```diff
@@ -6,35 +6,49 @@
...
+**/
 
 class SHT31 : public Sensor {
  public:
-  SHT31(uint8_t addr = 0x44, String config_path = "");
-  Adafruit_SHT31* adafruit_sht31;
-
- private:
-  uint8_t addr;
+  SHT31(uint8_t addr = 0x44);
```
Comment (joelkoz) [inline src/sensors/sht31.h:25]: Are you 100% sure there is no use case where someone would want to create a descendant class of SHT31?  If they did, and if they wanted to have THAT class be configurable, there is no way to pass the configuration path down to chain to the base classes.

**Review by joelkoz — COMMENTED** [review]
One comment about removing of config_path

**Review by joelkoz — APPROVED** [review]
Actually - I studied this code a little more, and I now understand that this design pattern is used elsewhere, and the class I had a question about is not really a configurable SendESP "thing" used directly, but is instead used by the other "Value" class.  It threw me that it inherited from "Sensor" yet it's not really meant to be observed directly.

**Comment by ba58smith** [comment]
Yeah, the constructor is really just a kind of wrapper for the Adafruit_SHT31 class, so it can be used like many other sensors. Actually, most of them have nothing configurable, so maybe none of them need to inherit from Sensor, do they? Should I go back and make them not descend from anything? (In a separate PR, of course.)

File: src/sensors/sht31.h (hunk 1 above)
Comment (ba58smith) [reply src/sensors/sht31.h:25]: See your other comment about this - you resolved it there.

**Pull request merged**